### PR TITLE
Announcement: Scala Center Advisory Board

### PR DESCRIPTION
**Pull request by larsrh**
New blog post.

### CODE REVIEW

**Comment by tpolecat** [comment]
This is likely to attract the attention of people who aren't that familiar or deeply involved with Typelevel, so it might be nice to introduce yourself in a sentence or two: *For those of you who don't know me…*

**Comment by larsrh** [comment]
@tpolecat There will be a blog post by @propensive on the scala-lang blog with basically the same announcement, but from the Scala Center perspective, including a short introduction of myself. I think that should be sufficient.

File: posts/2016-10-15-scala-center.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+>
+> It currently has seven voting members: representatives from each of our six sponsors, plus Bill Venners, the community representative. Additionally the Executive Director of the Scala Center, Heather Miller, sits on the board to report on the Scala Center’s activities, and provide advice on the feasibility of the proposals under consideration, and Martin Odersky is the technical advisor to the board.
+
+More details, including minutes and bylaws, can be found on [their website](https://scala.epfl.ch/).
+
+## Process
+
+To make the nomination as transparently as possible, I [opened an issue](https://github.com/typelevel/general/issues/42) shortly after Jon mailed me about the opening.
+The discussion period ended on October 12th.
+All candidates endorsed me, so no vote was necessary.
```
Comment (tpolecat) [inline posts/2016-10-15-scala-center.md]: s/candidates/nominees/?

Comment (larsrh) [inline posts/2016-10-15-scala-center.md]: Done.

**Comment by propensive** [comment]
LGTM.

**Comment by propensive** [comment]
By the way, I've not written my blog post yet, because I'm slow. Would you like to sync on the release?

**Comment by larsrh** [comment]
> Would you like to sync on the release?

Yes, please.

**Pull request merged**